deploy/helm/secret-operator/crds/crds.yaml

```diff
@@ -53,6 +53,10 @@ spec:
                           required:
                             - secret
                           type: object
+                        maxCertificateLifetime:
+                          default: 15days
```

Comment:
Do we want to support the long-form format here?

Reply:
We use https://docs.rs/humantime-serde/1.1.1/humantime_serde/, which in turn supports both - short and long format

Reply:
The question is not about whether we can or do, but about whether it's something that we want to support. Especially keeping in mind that it's much easier to relax the format later than to tighten it.

Reply:
That makes sense. Do you have an idea how we can easily achieve this? Should we not rely on humantime-serde but instead implement our own parser?

Reply:
If we want to use that then it should probably be moved into op-rs...

Reply:
Switched to pr to use the new Duration from operator-rs